resources/lib/npoapiclasses.py

```diff
@@ -4,6 +4,9 @@
 from datetime import datetime
 import time
 
+import xbmc
```

Comment:
Deze import is hier niet nodig, ik probeer alleen xbmc aan te spreken in de addon.py zodat ik de rest los kan aanroepen om te testen. Dat gaat een stuk fijner met debuggen